nfs/rpc/client.go

```diff
@@ -68,11 +72,14 @@ type message struct {
 }
 
 func (c *Client) Call(call interface{}) (io.ReadSeeker, error) {
+	retries := 1
```

Comment:
This can get hairy.  It's semantically correct, but I was afraid we'd retry atomic operations when really the unmarshaller was off by a few bytes or something nonsensical like that.  Anyway, just providing color.

Reply:
Yeah - I don't really like it but if I don't then we won't work with [certain IBM storage appliances](https://www.ibm.com/support/knowledgecenter/en/ST9SXX/com.ibm.storage.rtca.help.doc/tr_r_knowbase_GARBAGE_ARGS_disable.html) - the actual docs for that appliance contain more details, but relies on the retry.
Saying "we're not using the Linux client" gets awkward as a configuration input.